optuna/visualization/_contour.py

```diff
@@ -323,7 +325,17 @@ def _get_contour_subplot_info(
             value = target(trial)
         assert value is not None
 
-        z_values[(x_i, y_i)] = value
+        existing = z_values.get((x_i, y_i))
+        if existing is None or target is not None:
+            # In multi-objective case we don't know which objective was selected in target
+            # function therefore we don't know how to select the best objective value.
```

Comment:
Since `target` is also used in single-objective optimization, could you please modify this comment? For example, other metrics or elapsed time may be used as `target`.

Reply:
Good point, updated in 4d7f4077e99faeb2234868e450f8b1e1793c510f.